### ISSUE TITLE
Document Devcontainer setup

### ISSUE THREAD

**Issue by grossvogel** [revelrylabs/revelry_phoenix_app_template#958]
Let's add something to the readme about using the devcontainer setup, and have a few people test it to make sure it works as advertised.

**Comment by grossvogel** [comment]
Possible starting point:
----------------------------

## Running in a VS Code Dev Container

### Requirements
- Docker Desktop
- VS Code
- [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension for VS Code

### Usage
In VS Code, you can hit Cmd+Shift+P and type `Remote-Con...` to see commands pertaining to the devcontainer functionality. When you open a project with a devcontainer setup, the editor will prompt you to reopen in the container, or you can use the `Remote - Containers: Reopen in Container` command to do so manually.

Once the container is running, any terminal you open in the editor should run inside the container, where you can run all of the standard commands like `mix deps.get`, `mix ecto.migrate`, etc. The server will not start automatically, so you can do that from a terminal in the container with `bin/server` or `mix phx.server`.  The editor will prompt you again to open a browser tab pointing at the newly available site.

Beyond that, you can develop the app as usual by editing files in VS Code and seeing the changes reflected in the running site.

### Remote Database Container
... something about how to access the containerized database... we maybe also want to tweak the setup to expose a port on the host for people who use a database GUI?

**Comment by asp2131** [comment]
I cloned the project and reopened it within a remote container. I was able to run commands in the terminal once the container started running. I ran `mix deps.get`, `bin/setup`, and `bin/server` to grab all my dependencies, execute database migrations, and finally start the application. The application was running on http://localhost:4000. There was a `Port` tab on the terminal that enables you add extra ports if needed.
![image](https://user-images.githubusercontent.com/42776703/121933211-618e9280-cd0b-11eb-9f22-7a4ccaa1973b.png)
